Packages/com.unity.inputsystem/InputSystem/InputSettings.cs

```diff
@@ -301,6 +302,35 @@ public float defaultButtonPressPoint
             }
         }
 
+        /// <summary>
+        /// At what percentage of <see cref="defaultButtonPressPoint"/> a button that was pressed
+        /// is considered released again.
+        /// </summary>
+        /// <remarks>
+        /// This setting helps avoid flickering around the button press point by introducing something akin to a
+        /// "dead zone" below <see cref="defaultButtonPressPoint"/>. Once a button has been pressed to a magnitude
+        /// of at least <see cref="defaultButtonPressPoint"/>, it is considered pressed and then is considered pressed
```

Comment:
```suggestion
        /// of at least <see cref="defaultButtonPressPoint"/>, it is considered pressed and keeps being considered pressed
```

Reply:
Tried to make this a bit easier to parse; if it changes the meaning, please revert.

Reply:
Pushed